.gitignore

```diff
@@ -9,3 +9,4 @@
 /*.egg-info
 /htmlcov
 __pycache__
+.idea
```

Comment:
Unrelated change. Please make sure exclude your pyCharm related files via a [global .gitignor](https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore).
```suggestion
```

Reply:
ok, thank you

There is a flake8 addon, called flake8-unused-arguments. The name speaks for itself

Reply:
Trailing newline is missing. I'd suggest reviewing your own changeset once, to make sure nothing commit that you didn't intend to change.